js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,12 @@
+const animalSort = animals => {
+  return animals.sort((firstAnimal, secondAnimal) => {
+    if (firstAnimal.numberOfLegs === secondAnimal.numberOfLegs) {
```

Comment:
The result of the difference could have been stored and then used at line 3 and 8 as well.

```
const legsCountDifference = animal1.numberOfLegs - animal2.numberOfLegs;

if (legsCountDifference === 0) {
...
} else {
  return legsCountDifference;
}
```

Reply:
Updated with changes